src/dispatching/updater.rs

```diff
@@ -103,40 +102,35 @@ pub trait Updater:
     type Error;
 }
 
-#[pin_project]
-pub struct StreamUpdater<S> {
-    #[pin]
-    stream: S,
+type StreamItem = Result<Update, RequestError>;
+
+struct InnerUpdater<'a> {
```

Comment:
The structure is private, but you return it from `polling`.

Reply:
I'll fix it.